pixeltable/tests/conftest.py

```diff
@@ -38,10 +38,40 @@ def init_env(tmp_path_factory) -> None:
 
 @pytest.fixture(scope='function')
 def test_client(init_env) -> pt.Client:
+    # Clean the DB *before* instantiating a client object. This is because some tests
+    # (such as test_migration.py) may leave the DB in a broken state, from which the
+    # client is uninstantiable.
+    clean_db()
     cl = pt.Client(reload=True)
     cl.logging(level=logging.DEBUG, to_stdout=True)
     yield cl
-    cl.reset_catalog()
+
+
+def clean_db(restore_tables=True):
+    from pixeltable.env import Env
+    # (aaron-siegel) The logic from Client.reset_catalog() has been moved here, so that it
+    # does not rely on instantiating a Client object. As before, UUID-named data tables will
```

Comment:
uuid-named?

Also, the intention is to delete the data as well. I forgot why I commented it out.

It requires TableVersion.drop() in order to clean out media files as well.